### PR TITLE
fix login failure on interop tests with error: x509

### PR DESCRIPTION
**Pull request by anandrkskd**
Signed-off-by: anandrkskd <anandrkskd@gmail.com>

**What type of PR is this:**

/kind bug
/kind tests

**What does this PR do / why we need it:**
This PR fixes failing interop tests on 4.11/4.12.
Tests are failing as it doesn't contain changes from [PR](https://github.com/redhat-developer/odo/pull/6121) which was implemented on v2 branch.

**Which issue(s) this PR fixes:**

Fixes #NA

**PR acceptance criteria:**

- [ ] Unit test 

- [ ] Integration test 

- [ ] Documentation 

**How to test changes / Special notes to the reviewer:**

### CODE REVIEW

**Review by rm3l — COMMENTED** [review]
/approve

**Comment by rm3l** [comment]
@anandrkskd Can you rebase this branch onto `main`? Tests on Prow should be fixed with https://github.com/redhat-developer/odo/issues/6291

File: scripts/openshiftci-presubmit-all-tests.sh [hunk 1]
```diff
@@ -27,7 +27,7 @@ chmod 640 $TMP_DIR/kubeconfig
 export KUBECONFIG=$TMP_DIR/kubeconfig
 
 # Login as developer
-oc login -u developer -p password@123
+oc login -u developer -p password@123 --insecure-skip-tls-verify
```
Comment (rm3l) [inline scripts/openshiftci-presubmit-all-tests.sh:30]: I can see other occurrences of `odo login -u developer` in some of our scripts:
- [`scripts/openshiftci-Nightly-SBO-tests.sh`](https://github.com/redhat-developer/odo/blob/main/scripts/openshiftci-Nightly-SBO-tests.sh#L25)
- [`scripts/openshiftci-periodic-tests.sh`](https://github.com/redhat-developer/odo/blob/main/scripts/openshiftci-periodic-tests.sh#L24).

Do we need to update those too?

Comment (anandrkskd) [reply scripts/openshiftci-presubmit-all-tests.sh:30]: We can add this to other script as well, but it's mostly required for the interop env.
But as it does no harm, I will add it.

**Review by rm3l — APPROVED** [review]
LGTM - thanks.

Can you rebase this branch onto `main`? Tests should be fixed with https://github.com/redhat-developer/odo/issues/6291

**Comment by rm3l** [comment]
Tests passing on Prow, but not on IBM Cloud, with similar errors (`net/http: TLS handshake timeout`):

```
[odo]  •  Downloading starter project "nodejs-starter"  ...
    [odo]  ✗  Downloading starter project "nodejs-starter" [10s]
    [odo]  ✗  unable to download starter project "nodejs-starter": Get "[https://github.com/odo-devfiles/nodejs-ex.git/info/refs?service=git-upload-pack&quot;:](https://github.com/odo-devfiles/nodejs-ex.git/info/refs?service=git-upload-pack%22:) net/http: TLS handshake timeout
    [odo] the command failed, the devfile has been removed from current directory
    Deleting project: cmd-add-binding-test253onx
    Running kubectl with args [kubectl delete namespaces cmd-add-binding-test253onx --wait=false] and odo env: []
    [kubectl] namespace "cmd-add-binding-test253onx" deleted
    Setting current dir to: /go/odo_1/tests/integration
    Deleting dir: /tmp/1748382796
    Deleting dir: /tmp/2598565857
  << End Captured GinkgoWriter Output

  No future change is possible.  Bailing out early - after 14.022s.
  Running odo with args [odo init --name mynode --devfile-path /go/odo_1/tests/examples/source/devfiles/nodejs/devfile.yaml --starter nodejs-starter] and odo env: []
  Expected
      <int>: 1
  to match exit code:
      <int>: 0
  In [BeforeEach] at: /go/odo_1/tests/helper/helper_cmd_wrapper.go:101
```

Let's try again..

**Comment by rm3l** [comment]
/override OpenShift-Integration-tests/OpenShift-Integration-tests

E2E Tests failing on `net/http: TLS handshake timeout` when fetching starter project from GitHub. Previous run passed.

**Pull request merged**